### ISSUE TITLE
Ghost button is missing focus style

### ISSUE THREAD

**Issue by adamliptrot-oc** [w3c/w3c-website#403]
**Describe the issue**
The Ghost Button style does not have a visible focus indicator.

**URL**
https://design-system.w3.org/styles/buttons.html#ghost-button

**Recommended solution**
Add a visible focus indicator as https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html

**Comment by vivienlacourba** [comment]
cc: @iadawn

**Comment by iadawn** [comment]
+1 should have some focus indicator. Removing:

```
.button--ghost:focus {
  background-color: transparent;
  border-color: transparent;
  color: #000;
}
```

From core.css line 634 should do the job.

**Comment by NicolaSaunders** [comment]
Pull request to address this issue: https://github.com/w3c/w3c-website-templates-bundle/pull/125

**Comment by deniak** [comment]
The fix has been deployed.
